asio/CMakeUserPresets.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file should NOT be in version control: https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html

Reply:
I know, but the file get always lost with `git clean -xdf`